### DISCUSSION TITLE
Set browser page title with Browsershot

### DISCUSSION THREAD

**Discussion by SebKay** [spatie/browsershot#520] [General]
Apologies if there is an obvious solution to this but I've scoured the documentation and have come up short.

I'm trying to find a way of changing the browser page title (as in the title in a Chrome tab) when outputting a PDF to the page. I can't seem to find any method in Browsershot that will allow me to do this. I've tried the header and footer options, however these seemingly did nothing to the outputted PDF.

It's worth noting I had to add the following header before Browsershot is run to make sure it actually displays correctly (otherwise it's just a bunch of scrambled characters, like when you open a PNG in a text editor):

```php
header('Content-Type: application/pdf');
```

If anyone has a solution to adding the page title that would be greatly appreciated!

Thanks!

**Comment by SkiperX** [comment]
Faced the same problem. Does anyone have a solution?
